### PR TITLE
Fix brightness read from ACPI and its calculation

### PR DESCRIPTION
**Pull request by KFilipek**
This patch fixes path to specific ACPI device
and brightness calculation.

### CODE REVIEW

File: src/system_info/system_info_display_wayland.cc [hunk 1]
```diff
@@ -204,7 +204,7 @@ bool SysInfoDisplay::UpdateBrightness() {
   int val = atoi(str_val);
   free(str_val);
 
-  brightness_ = val / max_val;
```
Comment (rakuco) [inline src/system_info/system_info_display_wayland.cc]: Isn't this a fix for something else?

Comment (KFilipek) [inline src/system_info/system_info_display_wayland.cc]: Do you mean new patch for this one line change or extend commit message?

Comment (rakuco) [inline src/system_info/system_info_display_wayland.cc]: I'm asking what you had in mind. At first I thought this was a commit to fix the path name above, then I saw this change in the cast here that seems to fix something else related to the brightness code. It would be good to split this into two separate commits. And might even make more sense to just use `strtod()` instead of doing all those casts for this part.

Comment (halton) [inline src/system_info/system_info_display_wayland.cc]: Yes, I do not see why `static_cast<double>` is needed. Could you explain?

Comment (rakuco) [inline src/system_info/system_info_display_wayland.cc]: Well, it's needed because otherwise you're doing integer arithmetic (`val` and `max_val` are both `int`).

Comment (KFilipek) [inline src/system_info/system_info_display_wayland.cc]: Division of two integers give us integer so variable brightness_ is set only to 1 or 0. I think that expected brightness_ value is between 0 ... 1.

Comment (halton) [inline src/system_info/system_info_display_wayland.cc]: Right, thank for fixing the silly mistake when I wrote that code. @rakuco any concern left?

Comment (rakuco) [inline src/system_info/system_info_display_wayland.cc]: My only concern has always been that these are two separate fixes that could be committed separately (even if part of the same pull request). And if both `max_val` and `val` were `double`s (and then we use `strtod()` instead of `atoi()`) we could avoid the cast altogether.

**Comment by rakuco** [comment]
lgtm
